builder/openstack/step_key_pair.go

```diff
@@ -117,54 +106,10 @@ func (s *StepKeyPair) Run(ctx context.Context, state multistep.StateBag) multist
 
 	// Set some state data for use in future steps
 	s.Comm.SSHKeyPairName = s.Comm.SSHTemporaryKeyPairName
-	s.Comm.SSHPrivateKey = []byte(keypair.PrivateKey)
 
 	return multistep.ActionContinue
 }
 
-// Work around for https://github.com/hashicorp/packer/issues/2526
-func berToDer(ber []byte, ui packersdk.Ui) []byte {
```

Comment:
Looking at the linked issue on why this was added and seeing that the Openstack API is not the one creating the SSH key this workaround is no longer necessary, correct?

Reply:
Indeed, since openstack api can only generate ssh-rsa keys or x509. The go api let us push already existing keypairs https://github.com/gophercloud/gophercloud/blob/master/openstack/compute/v2/extensions/keypairs/requests.go#L89
The api doesn't even return content of the uploaded key then we do not parse openstack keys anymore

Reply:
Great thanks for the quick response.